src/router.js

```diff
@@ -3,78 +3,164 @@ import Router from 'vue-router'
 
 Vue.use(Router)
 
+// Base routes.
+const routes = [
+  {
+    path: '/',
+    name: 'Home',
+    component: () => import(/* webpackChunkName: "Home" */ './views/home/Index')
+  },
+  {
+    path: '/about',
+    name: 'About',
+    component: () =>
+      import(/* webpackChunkName: "About" */ './views/about/Index')
+  },
+  {
+    path: '/storybook-alike',
+    name: 'StorybookAlike',
+    component: () =>
+      import(
+        /* webpackChunkName: "StorybookAlike" */ './views/storybookalike/Index'
+      )
+  },
+  {
+    path: '/tags',
+    name: 'TagIndex',
+    component: () =>
+      import(/* webpackChunkName: "TagIndex" */ './views/tags/Index')
+  },
+  {
+    path: '/tags/:tagSlug',
+    name: 'TagShow',
+    component: () =>
+      import(/* webpackChunkName: "TagShow" */ './views/tags/Show')
+  },
+  {
+    path: '/categories',
+    name: 'CategoryIndex',
+    component: () =>
+      import(/* webpackChunkName: "CategoryIndex" */ './views/categories/Index')
+  },
+  {
+    path: '/category/:categorySlug',
+    name: 'CategoryShow',
+    component: () =>
+      import(/* webpackChunkName: "CategoryShow" */ './views/categories/Show')
+  },
```

Comment:
(dropping in some comments to highlight important changes, since lots of the changes on this branch are from prettier)

I added this section, which hooks up individual category pages